bin/prepare_rqc_simulation_files.jl

```diff
@@ -37,21 +30,35 @@ function parse_commandline(ARGS)
             default = true
             arg_type = Bool
         "--time"
-            help = "The number of seconds to run quickbb for."
-            default = 120
+            help = "The number of seconds to run flow cutter for."
+            default = 30
+            arg_type = Int64
+        "--sliced_bonds", "-s"
+            help = "Number bonds to slice"
+            default = 2
             arg_type = Int64
-        "--qbb_order"
-            help = "The branching order to be used by quickbb."
-            default = :min_fill
-            arg_type = Symbol
-        "--lb"
-            help = "Set if a lowerbound for the treewidth should be computed."
-            default = false
-            arg_type = Bool
         "--score_function"
             help = "Function to maximise when selecting vertices to remove."
             default = :direct_treewidth
             arg_type = Symbol
+        "--output_method"
+            help = "Method for selecting amplitudes to output. (:uniform, :rejection, :list)"
```

Comment:
Help is a little misleading as if `:` is included then this is also added to the symbol name

```suggestion
            help = "Method for selecting amplitudes to output. (uniform, rejection, list)"
```